src/List/List.jsx

```diff
@@ -0,0 +1,67 @@
+import _ from "lodash";
+import classnames from "classnames";
+import React, {PropTypes, PureComponent} from "react";
+
+import Item from "./Item";
+import MorePropTypes from "../utils/MorePropTypes";
+
+import "./List.less";
+
+const RowType = {
+  PLAIN: "plain",
+  BORDERED: "bordered",
+  ZEBRA: "zebra",
+};
+
+const cssClass = {
+  CONTAINER: "List",
+  EMPTY: "List--empty",
+  HEADER: "List--header",
```

Comment:
ah, the idea was to have PascalCase for top-level components (basically, match the javascript class name) - so far, we've been using camelCase for sub-elements and modifiers (i.e. `.List--Item` is a (sub-)component used with `List`, `.List--header` is a sub-element directly within `List`'s markup, `.List--header--small` would be a modified version of the header sub-element, etc.)